source/patterns/@aws-solutions-constructs/aws-s3-stepfunctions/lib/index.ts

```diff
@@ -110,15 +110,14 @@ export class S3ToStepfunctions extends Construct {
 
     if (!props.existingBucketObj) {
       [this.s3Bucket, this.s3LoggingBucket] = defaults.buildS3Bucket(this, {
-        bucketProps: props.bucketProps,
+        bucketProps: defaults.consolidateProps({ eventBridgeEnabled: true }, props.bucketProps),
```

Comment:
This is backwards, you've got the key prop for the construct as a default prop which can be overwritten. You need to make it the third argument, constructProps, which are props that the construct MUST turn on, so overwrite any customer specification.

defaults.consolidateProps({}, props.bucketProps, { eventBridgeEnabled: true });